swagger-coding-style.md

```diff
@@ -244,15 +244,15 @@ about the operation behavior -- it should not simply restate the summary.
 
 Every operation should have a unique `operationId`.
 In the Swagger specification, `operationId`s are optional, but if specified, must be unique.
-For SDK generation, `operationId`s are used as the name of the method corresponding to the operation, so it is important to include these in the swagger.
+For SDK generation, `operationId`s are used as the name of the method corresponding to the operation, so it is important to include these in the OpenAPI definition file.
 
 The `operationId` should be specific and descriptive. Here is the recommended convention:
 
-    - GET a single `Model`: `getModel`
-    - GET a list of `Model`: `listModels`
-    - POST a new `Model`: `createModel` or `addModel`
-    - PUT an update to a `Model`: `updateModel`
-    - DELETE a `Model`: `deleteModel`
+- GET a single `Model`: `getModel`
+- GET a list of `Model`: `listModels`
+- POST a new `Model`: `createModel` or `addModel`
+- PUT an update to a `Model`: `updateModel`
```

Comment:
We should distinguish between a "selective update" and "complete replacement", so this line should be replaced with:
```
- POST a partial update to a `Model`: `updateModel`
- PUT a complete replacement to a `Model`: `replaceModel`
```

Reply:
Done